File: PRAK110-2519817320004-RahmaSyaritaJaya.c
#include <stdio.h>
#include <math.h>

int main() {
    int a = 5, b = 12, c = sqrt((a*a)+(b*b));
    printf("Diketahui:\nAlas = %d cm\nTinggi = %d cm\n\n", a,b);
    printf("Jawab:\nSisi A = %d cm\nSisi B %d cm\nSisi C = %d cm\n", a,b,c);
    printf("Keliling = %d cm\nLuas = %d cm", a+b+c, (a*b)/2);
    return 0;
}

File: PRAK110-2519817320004-RahmaSyaritaJaya.py
import math

a, b = 5, 12
c = int(math.sqrt((a*a)+(b*b)))

print(f"Diketahui:\nAlas = {a} cm\nTinggi = {b} cm\n")
print(f"Jawab:\nSisi A = {a} cm\nSisi B {b} cm\nSisi C = {c} cm")
print(f"Keliling = {a+b+c} cm\nLuas = {int((a*b)/2)} cm")